### ISSUE TITLE
[BUG] UI test runner crashes when an incorrect css selector is encountered

### ISSUE THREAD

**Issue by ondracja** [microsoft/playwright#21961]
### System info
- Playwright Version: [v1.32.0]
- Operating System: [macOS 13.2.1]
- Browser: [All]
- Other info:
  - node: [v14.18.1]
  - yarn: [1.22.18]

### Source code

```js
import { test } from '@playwright/test';

test('breaks the testrunner', async ({ page }) => {
    await page.goto('https://www.bing.com');
    await page.locator('[type="search').type('test');
});
```

**Steps**

1.  start the test runner via `npx playwright test --ui`
2. execute the test 'breaks the testrunner'

**Expected**

Test case fails due to an incorrect css selector.

**Actual**

The whole test runner crashes as seen in the attached screenshot.

<img width="2078" alt="image" src="https://user-images.githubusercontent.com/9817938/227601824-ecc1fc43-cfea-443b-8ecb-f6bdeed7a827.png">

**Comment by pavelfeldman** [comment]
Folding into https://github.com/microsoft/playwright/issues/21961

**Comment by pavelfeldman** [comment]
This will be fixed in 1.32.1
